tests/test_streaming_services.py

```diff
@@ -0,0 +1,42 @@
+import time
+
+from util.solver_workflow import new_solver_session  # noqa: F401
+
+from ansys.fluent.core.fluent_connection import _FluentConnection
+from ansys.fluent.core.session_solver import Solver
+
+
+def transcript(data):
+    transcript.data = data
+
+
+def run_transcript(i, ip, port, password):
+    solver_session = Solver(_FluentConnection(ip=ip, port=port, password=password, cleanup_on_exit=False))
+    solver_session.transcript.register_callback(
+        transcript
+    )
+
+    if i % 5 == 0:
+        solver_session.scheme_eval.scheme_eval("(pp 'test)")
+        check_transcript = True
+        time.sleep(1)
+    else:
+        check_transcript = False
+
+    if solver_session:
+        solver_session.exit()
+        if check_transcript:
+            if not transcript.data:
+                print(i, 'transcript failed.', transcript.data)
```

Comment:
assert?

Reply:
I guess that @prmukherj knows that this trips up and doesn't want to merge a failing test.

Reply:
@ajain-work, @seanpearsonuk, we can replace the print with assert once we have the server side fix. Sounds all right?

Reply:
Yes, I agree.